src/Admin/Models/OrganizationViewModel.cs

```diff
@@ -63,4 +64,5 @@ public class OrganizationViewModel
     public int Projects { get; set; }
     public int ServiceAccounts { get; set; }
     public int SmSeats { get; set; }
+    public bool UseSecretsManager { get; set; }
```

Comment:
You can access this via the existing `Organization` property without adding a new property. You can refer to it via a getter for convenience if you want though.

Reply:
resolved